README.md

```diff
@@ -189,6 +189,55 @@ $ image-builder list-images --output=json
 ]
 ```
 
+## Overriding the base repositories
+
+There are various ways to add extra repositories or override the default
+base repositories. Most users will want to use the
+[blueprint systems](https://osbuild.org/docs/user-guide/blueprint-reference/#repositories)
+for this. Repositories that are part of the blueprint will get added
+to the installed image but are not used at build time to install third-party
+packages.
+
+To change repositories during image build time the commandline options
+`--datadir`, `--add-repo` and `--force-repo` can be used. The repositories
+there will only added during build time and will not be available in the
+installed system (use the above blueprint options if that the goal).
+
+Note that both options are targeting advanced users/use-cases and when
+used wrongly can result in failing image builds or non-booting
+systems.
+
+## Using the datadir switch
+
+When using the `--datadir` flag `image-builder-cli` will look into
```

Comment:
I must admit that I didn't thoroughly check the rest of the docs, but shouldn't we refer to the _binary name_? So `image-builder` vs. `image-builder-cli`.

Reply:
Thanks, nice catch! I think just here, fixing